src/utils/feeEstimate.js

```diff
@@ -0,0 +1,18 @@
+import {ethers} from "ethers";
+import {get} from "svelte/store";
+import * as VeFeeEstimateABI from "./abis/veFeeEstimateABI";
+import { networkSigner } from "../stores/web3";
+
+const veFeeEstimateABI = VeFeeEstimateABI.default
+
+export const getRewardsFeeEstimate = async(userAddress) => {
+    try {
+      const contract = new ethers.Contract(process.env.VE_FEE_ESTIMATE_CONTRACT, veFeeEstimateABI, get(networkSigner));
+      const estimateClaim = await contract.estimateClaim(userAddress)
+      const estimateClaimFormatted = ethers.utils.formatEther(BigInt(estimateClaim).toString(10))
+      return estimateClaimFormatted
+  } catch (error) {
+    console.log(error)
+    throw error;
+  }
+}
```

Comment:
We could also move this content over to a common rewards file since it would be just one function in this file

Reply:
I'm going to suggest we leave this like this, because the `src/utils/` file-pattern is to maps fns 1:1 w/ the ABI